test/jdk/javax/swing/JColorChooser/Test4759306.java

```diff
@@ -21,22 +21,37 @@
  * questions.
  */
 
+import javax.swing.JColorChooser;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+
 /*
  * @test
  * @bug 4759306
+ * @library /test/jdk/java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
  * @summary Checks if JColorChooser.setPreviewPanel removes the old one
- * @author Konstantin Eremin
-   @run applet/manual=yesno Test4759306.html
+ * @run main/manual Test4759306
  */
+public class Test4759306 {
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                .title("Test4759306")
+                .instructions("Check that there is no preview panel in the JColorChooser.")
```

Comment:
> Check that there is no preview panel

For a tester who didn't use JColorChooser before this can be confusing

It might be worth describing where to look for the Preview panel and what it looks like(e.g. `Check that there is no panel titled "Preview"` or similar)